backend/README.md

```diff
@@ -1,4 +1,60 @@
 # Pizza Backend
+
 ## Design
 
-We are using Go-lang for the backend. More specifically the [Gin](https://github.com/gin-gonic/gin) web framework for the API, and plans for [Gorilla](https://github.com/gorilla/websocket) as the web socket.
\ No newline at end of file
+We are using Go-lang for the backend. More specifically the [Gin](https://github.com/gin-gonic/gin) web framework for the API, and plans for [Gorilla](https://github.com/gorilla/websocket) as the web socket.
+
+## Routes
+
+We will require routes for the user to be able to enroll into classes, make posts, reply to posts, etc. These should all be grouped under a user route, where we use either the users name or id
+in the route itself for the ability to "enroll" them into a class.
+
+### Structure
+
+user_router := router.Group("/user"){
+
+    user_router.POST("/register", handleUserRegister) // The request respondst to the url matching "/user/register" and should include
```

Comment:
move this to the next line, it's a bit difficult to read. same for the other routes!